jmix-reports/reports-flowui/src/main/resources/io/jmix/reportsflowui/view/parameter/report-parameter-detail-view.xml

```diff
@@ -0,0 +1,156 @@
+                                  required="true"
+                                  dataContainer="parameterDc"
+                                  property="predefinedTransformation"
+                                  width="100%"/>
+                    </vbox>
+                    <div id="transformationEditorBox">
+                        <div classNames="flex, justify-between">
+                            <label text="msg://parameters.groovyScript"/>
+                            <button id="fullScreenTransformationBtn" text="msg:///fullScreenBtn.title"
+                                    themeNames="tertiary-inline"/>
+                        </div>
+                        <div classNames="flex, items-start">
+                            <codeEditor id="transformationScript"
+                                        width="100%"
+                                        height="15em"
+                                        mode="GROOVY"
+                                        property="transformationScript"
+                                        dataContainer="parameterDc"/>
+                            <button id="transformationScriptHelpBtn"
+                                    themeNames="icon, tertiary-inline"
+                                    icon="QUESTION_CIRCLE"/>
+                        </div>
+                    </div>
+                </div>
+            </tab>
+            <tab id="validationTab" label="msg://parameters.validationTab">
+                <div id="includeValidationBox">
+                    <checkbox id="includeValidation" dataContainer="parameterDc"
+                              label="msg://parameters.validateFields"
+                              property="validationOn"/>
+                    <div classNames="flex, justify-between">
+                        <label text="msg://parameters.groovyScript"/>
+                        <button id="fullScreenValidationBtn" text="msg:///fullScreenBtn.title"
+                                themeNames="tertiary-inline"/>
+                    </div>
+                    <div classNames="flex, items-start">
+                        <codeEditor id="validationScript"
+                                    width="100%"
+                                    height="15em"
+                                    mode="GROOVY"
+                                    property="validationScript"
+                                    dataContainer="parameterDc"/>
+                        <button id="validationScriptHelpBtn"
+                                themeNames="icon, tertiary-inline"
+                                icon="QUESTION_CIRCLE"/>
+                    </div>
+                </div>
+            </tab>
+            <tab id="lookupSettingsTab" label="msg://parameters.dropdownSettingsTab">
+                <div id="lookupSettingsBox" width="100%">
+                    <div classNames="flex, justify-between">
+                        <label text="msg://lookupJoin"/>
```

Comment:
Since it a `label`, it needs `setFor` attribute. If it's just a test, may be worth using another element, like `span`